src/sentry/hybridcloud/services/region_organization_provisioning/impl.py

```diff
@@ -52,8 +52,9 @@ def _create_organization_and_team(
         assert (user_id is None and email) or (
             user_id and email is None
         ), "Must set either user_id or email"
+        truncated_name = organization_name[:64]
```

Comment:
Should we have a constant in `sentry.models.organization` that defines this length?

Reply:
That's a good idea, I'll add one.